lib/pos_blockchain.go

```diff
@@ -968,6 +968,50 @@ func (bc *Blockchain) getHighestCommittedBlock() (*BlockNode, int) {
 	return nil, -1
 }
 
+func (bc *Blockchain) GetSafeBlocks() ([]*BlockNode, error) {
+	// First get committed tip.
+	committedTip, idx := bc.getHighestCommittedBlock()
+	if idx == -1 || committedTip == nil {
+		return nil, errors.New("GetSafeBlocks: No committed blocks found")
+	}
+	// Now get all blocks from the committed tip to the best chain tip.
+	safeBlocks := []*BlockNode{committedTip}
+	maxHeightWithSafeBlocks := bc.getMaxSequentialBlockHeightAfter(uint64(committedTip.Height))
+	for ii := uint64(committedTip.Height + 1); ii < maxHeightWithSafeBlocks+1; ii++ {
+		blockNodes, exists := bc.blockIndexByHeight[ii]
```

Comment:
Also fine. Definitely safer to have a wrapper function like in to fetch all blocks at a certain height like in: https://github.com/deso-protocol/core/pull/785/files#r1383546779